src/main.rs

```diff
@@ -97,7 +97,7 @@ fn main() {
 
     match probe(&client) {
         Ok(()) => {}
-        Err(err) => println!("{}", err),
+        Err(err) => err.abort(None),
```

Comment:
Maybe an unwind would be better here.

Reply:
`abort()` here is a method of `NetBoxAPIError` struct, which prints the error and exits the programm with a given, non-zero error code.
This is used to avoid having to panic with fixable, user-caused errors.